src/crystal/project/dsl.cr

```diff
@@ -13,5 +13,9 @@ struct Crystal::Project::DSL
     def github(repository, name = nil : String)
       @project.dependencies << GitHubDependency.new(repository, name)
     end
+
+    def local(path, name = nil : String)
```

Comment:
Sure, no problem. Do you want me to submit another pull request, or you'll change it directly?